src/app/pages/resources/overview/overview.jsx

```diff
@@ -23,38 +24,54 @@ import ApiWrapper from '../../../api/api-wrapper';
 import theme from '../../../themes/theme';
 
 const useStyles = makeStyles({
-  pageHeader: {
-    padding: '32px',
+  content:{
+    display:'grid',
+    gap:'8px',
+    overflowWrap:'anywhere',
+  },
+  fabContainer: {
+    position: 'fixed',
+    zIndex: 1500,
+    bottom: '9em',
+    right: '1em',
+  },
+  fullWidth: {
+    gridColumnEnd: 'span 2',
+  },
+  iconSpacing: {
+    marginLeft: '4px',
   },
   pageBody: {
     display: 'grid',
-    gap: '16px',
+    gap: '32px',
     gridTemplateColumns: '1fr',
     padding: '32px',
     backgroundColor: '#f9f9f9',
     [theme.breakpoints.up('md')]: {
-      gridTemplateColumns: '1fr 4fr',
+      gridTemplateColumns: 'auto 4fr',
```

Comment:
If the first column is auto width, then the second can't be fractionally 4, right? Don't the fractional ones need to have something to compare against?